spec/features/login_spec.rb

```diff
@@ -30,3 +30,57 @@
     end
   end
 end
+
+describe 'login via email/password' do
+  before :each do
+    visit '/'
+    expect(page).to have_content 'Please login first'
```

Comment:
Da bin ich ehrlich gesagt kein Fan von, weil I18n für Use Cases gedacht ist, in denen die Sprache oft gewechselt wird. In Tests ist das aber nicht der Fall, da Rspec bei uns standardmäßig nur auf Englisch testet. Da I18n somit nicht genutzt werden würde und es den Test nur unleserlicher macht, würde ich gern beim englischen Value bleiben.